### PR TITLE
Update deltachat-node to v1.35.0 (async core)

### LINKED ISSUE #1215: Backup of larger DB freezes desktop for a minute
Importing a large DB (1.5GB) freezes Elektron/the desktop while core is starting the backup.  The backup is, however, happening in the imap-thread, and not i think with a sync-call to core.  So the blocking on Desktop is probably for other reasons -- is it maybe blocking on waiting for the first CONFIGURATION_PROGRESS event?  

Versions: On core 1.0.0-beta.15  current node and desktop/master 82108558f978109b6cc5b71203208a6e39f8b8d3

### LINKED ISSUE #1236: No progressbar on exporting backup

### PR DESCRIPTION
**Pull request by Jikstra**
- [x] release new -node version  
- [x] fix login
- [x] login progress bar is missing

closes #1236
closes #1215

Fixed exporting backup:
fix-export-backup

### CODE REVIEW

File: src/renderer/components/dialogs/Settings-Appearance.tsx [hunk 1]
```diff
@@ -128,8 +128,15 @@ class BackgroundSelector extends React.Component {
...
 export default function SettingsAppearance({
+<<<<<<< HEAD
   handleDesktopSettingsChange,
 }: {
+=======
+  forceUpdate,
+  handleDesktopSettingsChange,
+}: {
+  forceUpdate: any
+>>>>>>> 224aa6d8... We are starting with async core
```
Comment (Simon-Laux) [inline src/renderer/components/dialogs/Settings-Appearance.tsx:139]: a commited merge conflict :eyes: ?

File: src/renderer/components/dialogs/Settings-Backup.tsx [hunk 2]
```diff
@@ -1,12 +1,64 @@
...
-import { confirmationDialogLegacy as confirmationDialog } from './ConfirmationDialog'
 import { OpenDialogOptions } from 'electron'
 import { ipcBackend } from '../../ipc'
+import { DialogProps } from './DialogController'
+import DeltaDialog from './DeltaDialog'
+import { isOpen } from '@blueprintjs/core/lib/esm/components/context-menu/contextMenu'
 
 const { remote } = window.electron_functions
 
+function ExportProgressDialog(props: DialogProps) {
```
Comment (Simon-Laux) [inline src/renderer/components/dialogs/Settings-Backup.tsx:19]: nice, this will close #1236

**Comment by Simon-Laux** [comment]
this will probably close also #1215

**Comment by Simon-Laux** [comment]
BUG: I get blueprint warnings and the progressbar doesn't update on export.

**Comment by Simon-Laux** [comment]
somehow the integration tests don't like this pr

**Comment by Jikstra** [comment]
> please fix the export progressbar bug before merging

This was already broken before and i still didn't find the bug. I have the feeling that the progress events are all received at the end when it's already done and something is blocking the backend.

**Comment by link2xt** [comment]
I get a lot of warnings
```
core/event: DC_EVENT_WARNING 0 deltachat-ffi/src/lib.rs:1442: dc_get_msg called with special msg_id=0, returning empty msg
```

**Pull request merged**